src/content/docs/es/recipes/i18n.mdx

```diff
@@ -0,0 +1,333 @@
+
+    <StaticSsrTabs>
+      <Fragment slot="static">
+        En el modo de renderizado estático, utiliza `getStaticPaths` para asignar cada entrada de contenido a una página:
+
+        ```astro
+        //src/pages/[lang]/blog/[...slug].astro
+        ---
+        import { getCollection } from 'astro:content'
+
+        export async function getStaticPaths() {
+          const pages = await getCollection('blog')
+
+          const paths = pages.map(page => {
+            const [lang, ...slug] = page.slug.split('/');
+            return { params: { lang, slug: slug.join('/') || undefined }, props: page }
+          })
+
+          return paths;
+        }
+
+        const { lang, slug } = Astro.params;
+        const page = Astro.props;
+        const formattedDate = page.data.date.toLocaleString(lang);
+
+        const { Content } = await page.render();
+        ---
+        <h1>{page.data.title}</h1>
+        <p>by {page.data.author} • {formattedDate}</p>
+        <Content/>
+        ```
+      </Fragment>
+
+      <Fragment slot="ssr">
+        En el [modo SSR](/es/guides/server-side-rendering/), obtén directamente la entrada solicitada:
+
+        ```astro
+        //src/pages/[lang]/blog/[...slug].astro
+        ---
+        import { getEntryBySlug } from 'astro:content';
+
+        const { lang, slug } = Astro.params;
+        const page = await getEntryBySlug('blog', `${lang}/${slug}`);
+
+        if (!page) {
+          return Astro.redirect('/404');
+        }
+
+        const formattedDate = page.data.date.toLocaleString(lang);
+        const { Content, headings } = await page.render();
+        ---
+        <h1>{page.data.title}</h1>
+        <p>by {page.data.author} • {formattedDate}</p>
+        <Content/>
+        ```
+      </Fragment>
+    </StaticSsrTabs>
+
+    📚 Lee más sobre [enrutamiento dinámico](/es/core-concepts/routing/#rutas-dinámicas).
+
+    :::tip[Formateo de fechas]
+    El ejemplo anterior utiliza el método integrado [`toLocaleString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString) de formateo de fechas para crear una cadena legible por humanos a partir de la fecha del encabezado.
+    Esto asegura que la fecha y la hora se formateen de acuerdo con el idioma del usuario.
+    :::
+
+### Traducir strings de la UI
+
+Crea diccionarios de términos para traducir las etiquetas de los elementos de la UI en tu sitio. Esto permitirá que tus visitantes experimenten tu sitio completamente en su idioma.
+
+1. Crea un archivo `src/i18n/ui.ts` para almacenar tus strings de traducción:
+
+    ```ts
+    // src/i18n/ui.ts
+    export const languages = {
+      en: 'English',
+      fr: 'Français',
+    };
+
+    export const defaultLang = 'en';
+    
+    export const ui = {
+      en: {
+        'nav.home': 'Home',
+        'nav.about': 'About',
+        'nav.twitter': 'Twitter',
+      },
+      fr: {
+        'nav.home': 'Acceuil',
```

Comment:
Updated translation from https://github.com/withastro/docs/pull/3623
```suggestion
        'nav.home': 'Accueil',
```

Reply:
I'm gonna merge the open PR for this, then update this branch, so this will get taken care of automatically!